### PR TITLE
Add BEM (BEP20)

### PR DESCRIPTION
**Pull request by ThanhIres**
Adding info for BEP20 token 'BEM'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by ThanhIres** [comment]
https://github.com/trustwallet/assets/pull/18085
https://explorer.binance.org/tx/A196F32CF8D57AB247CDF3DB65AD3EA1A0CE678234E80C46D757B6ECE35F7966
We've made payment for this, but unlucky it was closed as my team took so long to processed the payment.
It's 5 BNB. Pls check it

**Comment by Cryptocool1** [comment]
Снимок экрана 2022-03-07 в 11 04 45
Hello, your logo should be full screen size

**Comment by ThanhIres** [comment]
Holders: 13,373 addresses

**Comment by ThanhIres** [comment]
> 
logo

**Comment by ThanhIres** [comment]
Our full screen size logo

*Thanks and Regards !*

*Nguyen Trung Thanh*

*Mobile: +84 984223092*
Email: ***@***.***
Skype: ***@***.***



Vào Th 2, 7 thg 3, 2022 vào lúc 10:05 CryptoA ***@***.***>
đã viết:

**Review by Bmg00 — COMMENTED** [review]
https://bscscan.com/token/0x7B86b0836f3454e50C6F6a190cd692bB17da1928

File: blockchains/smartchain/assets/0x7B86b0836f3454e50C6F6a190cd692bB17da1928/info.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (Bmg00) [inline blockchains/smartchain/assets/0x7B86b0836f3454e50C6F6a190cd692bB17da1928/info.json:1]: Repo

**Pull request merged**